.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,15 @@
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}/**"
+            ],
+            "defines": [],
+            "compilerPath": "/usr/bin/clang",
```

Comment:
Will VS Code figure out the clang path if it's not specified?

Reply:
I removed that line `"compilerPath": "/usr/bin/clang"` and VSCode start using `/usr/bin/c++`.